core/src/types/capability.rs

```diff
@@ -175,7 +175,7 @@ pub struct Capability {
     /// Indicates if recursive listing is supported.
     pub list_with_recursive: bool,
     /// Indicates if versioned listing is supported.
-    pub list_with_version: bool,
+    pub list_with_versioned: bool,
```

Comment:
Capability is part of opendal's public API so we can't simplely change it. Please add a new field and deprecate the old one.